packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledSection.tsx

```diff
@@ -236,7 +237,60 @@ export const StyledPropertiesAlarmsSection: FC<
         })
       ) ?? [];
 
-    const components = [...modeled, ...unmodeled, ...assetModeled];
+    const alarms =
```

Comment:
nit: there's probably a way for us to reuse the code between the section and styledSection files, but for now this is fine